include/path_tracking_pid/controller.hpp

```diff
@@ -96,22 +96,21 @@ class Controller : private boost::noncopyable
     const geometry_msgs::Twist & steering_odom_twist,
     const std::vector<tf2::Transform> & global_plan);
 
+  /** Result of findPositionOnPlan(). */
+  struct FindPositionOnPlanResult
+  {
+    tf2::Transform position;
```

Comment:
I'd rather see this member named `pose` instead of `position`, as it represents not only a position, but also the orientation of the projection onto the path (together, this is called the "pose"). Maybe this also implies that the function should be named `FindPoseOnPlan` or something. Does that make sense, @cesar-lopez-mar ?

Reply:
Indeed, also the data type represents a pose indirectly, we use transforms because we use functions of tf2 objects to make operations on them. When track_base_link is false we do not use the orientation but the same data type is used.

Reply:
Fixed in #144 to avoid merge conflicts in the PR train. Since the name was not changed yet, it could be considered just another refactor step.

Reply:
Sounds fair enough